getting-started.md

```diff
@@ -10,21 +10,17 @@ permalink: /getting-started/
 
 ### Plugin in Spring Boot MVC application
 
-In this getting started tutorial we will go though the process of integrating Nixer plugin into the simple Spring MVC application. If however you want to see necessary changes right away look at the diff in examples repository. Branch master contains application only and branch todo contains all the necessary modification to the codebase to have the plugin working.
+In this getting started tutorial we will go though the process of integrating Nixer plugin into the simple Spring MVC application. If however you want to see necessary changes right away look at [the diff in examples repository](https://github.com/nixer-io/nixer-spring-plugin-integrations/compare/with%2Fnixer-plugin-getting-started?diff=split). Branch _master_ contains application only and branch _with/nixer-plugin-getting-started_ contains all the necessary modification to the codebase to have the plugin working.
```

Comment:
Both branch names, master and with/nixer-plugin-getting-started, should rather be put in backticks for readability and consistency.

Reply:
I don't want to distract from what is important (at this point a user kthat eeps reading) with branch names. Also, I don't think it necessarily adds readability. 
I think context can change what is style ` __ - is suitable, not strict rules.